trio/ssl.py

```diff
@@ -4,5 +4,36 @@
 # temporaries, imports, etc. when implementing the module. So we put the
 # implementation in an underscored module, and then re-export the public parts
 # here.
-from ._ssl import *
-from ._ssl import __all__
+
+from ._ssl import (SSLStream, SSLListener, aclose_forcefully, ConflictDetector)
+
+# Fake import to enable static analysis tools to catch the names
+try:
+    from ssl import (
+        SSLError, SSLZeroReturnError, SSLSyscallError, SSLEOFError,
+        CertificateError, create_default_context, match_hostname,
+        cert_time_to_seconds, DER_cert_to_PEM_cert, PEM_cert_to_DER_cert,
+        get_default_verify_paths, Purpose, enum_certificates, enum_crls,
+        SSLSession, VerifyMode, VerifyFlags, Options, AlertDescription,
+        SSLErrorNumber
+    )
```

Comment:
These don't need to be fake! It's a well-defined set of things that should always be available. So we can just do a regular old `import` without any `try`/`except`.

Reply:
Some actually don't but others do. I tried to import them directly but the test failed on some versions of python and with windows. I decided to revert back to being them all fake since doing a subset one way and the rest the other is more confusing and IMHO not worth the extra lines of code. Is this fine with you?